fe2-android/app/src/test/java/com/github/dedis/popstellar/model/objects/security/KeyPairTest.java

```diff
@@ -44,4 +44,19 @@ public void getPublicKeyReturnsRightValue() {
 
     assertEquals(publicKey, keyPair.getPublicKey());
   }
+
+  @Test
+  public void pubKeyHash() {
+    // Tested with value in keypair.json (see #1042)
```

Comment:
ideally it would be better to pull the value from the file programmatically. (ofc requires merging in #1043)

Reply:
All others test are hardcoded yet, I mean the example will still true, so as there are not a lot of value I don't think this is a problem.

Reply:
ok fair enough. We aren't in the "ideally" realm anyways.